src/Components/01.Homepage/AboutUs.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+interface Props {
+}
+
+const AboutUs: React.FC<Props> = () => {
+  const header = 'What is Pinch?';
+
+  return (
+    <div className="AboutUs">
+      <img src="/assets/images/noimageavailable.jpg" alt="Visual of the front page" className="ABimg" />
```

Comment:
this class name also shouldn't start with capital letters if you can rename it to follow a hyphen standard or just be one word